### PR TITLE
add patch for BLIS 0.8.1 to fix dgemm FPE signalling on Broadwell

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

This fixes https://github.com/flame/blis/issues/486

### CODE REVIEW

File: easybuild/easyconfigs/b/BLIS/BLIS-0.8.1_fix_dgemm-fpe-signalling-on-broadwell.patch [hunk 1]
```diff
@@ -0,0 +1,2219 @@
+Taken from https://github.com/flame/blis/pull/544
+Fixes a problem with DGEMM causing FPR signalling on Broadwell
+See https://github.com/flame/blis/pull/544
```
Comment (boegel) [inline easybuild/easyconfigs/b/BLIS/BLIS-0.8.1_fix_dgemm-fpe-signalling-on-broadwell.patch:3]: Do we need to refer to the PR twice?

I guess you intended to refer to https://github.com/flame/blis/issues/486 here?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by akesandgren** [comment]
This now passes BLAS/TESTING as per https://github.com/easybuilders/blis-eval/tree/main/ake/blas-correctness-test on Broadwell without dgemm causing FPE signalling.
And still passes on SkylakeX and Zen2

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/3ae38ff9f5f19f670695ba33a2aa179b for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/54c835b1fc5e7429899aa245417f610a for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jrc0727 - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/754968da56d3a669a3edbb06ca2be937 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jwc09n000.juwels - Linux centos linux 8.4.2105, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/0e27cb1877b35c1090aa4039326dc2b3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3511.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/0ddc6ee73fa1aeab990e44f07d79a773 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2635.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/170ba2487cd47a874de5ba3571a16241 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**